stores/localeStore.ts

```diff
@@ -0,0 +1,28 @@
+import { defineStore } from 'pinia'
+
+const localeEn:{label: string, code: string} = {
```

Comment:
As a future enhancement (in a later PR) let's consider making a proper type rather than an anonymous type